### DISCUSSION TITLE
Cron en minute

### DISCUSSION THREAD

**Discussion by OUARZA** [OUARZA/Propluvia#52] [Ideas]
Permettre les minutes. En effet, à xh00, il y a déjà pas mal de tâche cron qui tourne sur les Jeedom, à savoir toutes les cron5, cron10, cron 1h, … Donc, perso j’aime bien varier et faire tourner sur des créneaux peu utilisés, comme 0h37 (par exemple) pour éviter les pics de charge inutile.

==> En effet, je sais pas si il est possible de mettre un cron aléatoire, je crois qu'ils le font côté netatmo, ainsi tout le monde n'interroge pas l'api au même moment du coup:
- mettre un menu déroulant pour heure et minute
- mettre par défaut l'heure sur 6
- mettre un chiffre aléatoire sur les minutes qui ne terminent ni par un 0 ni par un 5

**Comment by elmerdu69** [comment]
Passage en cron à la minute poussé sur la beta avec menu sélection des heures et minutes
![image](https://github.com/OUARZA/Propluvia/assets/115868585/c1952996-f2df-4f5e-858d-2e42022766a9)

je n'ai pas fixé d'heure par défaut ni minute aléatoire car complexe.
Par contre si la configuration n'est pas faite, cad sélection de l'heure et de la minute puis sauvegarde de la config, un message d'erreur s'affichera pour signaler que ça n'a pas été fait. Il sera répété à chaque passage dans le cron (chaque minute)

**Reply by OUARZA** [reply]
Toutes les minutes, ça ne fait pas un peu beaucoup ? :D

**Reply by elmerdu69** [reply]
Bah c'est le principe du cron minute 😅
Et puis ça forcera à faire une config correcte

**Comment by OUARZA** [comment]
Problème signalé de la redondance de l'alerte "L’heure de relevé n’a pas été correctement configurée dans la page de configuration du plugin" lorsqu'on laisse 00 minutes même si on change l'heure.
https://community.jeedom.com/t/message-derreur-sur-heure-de-releve/113634?u=ouarza

Peut-être revoir la condition de l'alerte ?
Est-il possible de mettre par défaut "rien" ?
